test/k8sT/Policies.go

```diff
@@ -376,6 +379,87 @@ var _ = Describe("K8sValidatedPolicyTest", func() {
 				helpers.CurlFail(fmt.Sprintf("http://%s/public", clusterIP)))
 			Expect(err).Should(HaveOccurred())
 		})
+
+		It("Denies traffic with k8s default-deny policy", func() {
+			namespace := "default"
+
+			kubectl.Apply(demoPath).ExpectSuccess("Creating demo app")
+			defer kubectl.Delete(demoPath).ExpectSuccess()
+
+			ciliumPod, err := kubectl.GetCiliumPodOnNode(helpers.KubeSystemNamespace, helpers.K8s1)
+			Expect(err).Should(BeNil())
+
+			pods, err := kubectl.WaitforPods(
+				namespace,
+				"-l zgroup=testapp", 300)
+			Expect(pods).To(BeTrue(), "testapp pods are not ready after timeout")
+			Expect(err).To(BeNil(), "testapp pods are not ready after timeout")
+
+			By("Testing connectivity without any policy loaded")
+			clusterIP, _, err := kubectl.GetServiceHostPort(namespace, "app1-service")
+			Expect(err).To(BeNil(), "Cannot get service on %q namespace", namespace)
+			appPods := getAppPods(namespace)
+			_, err = kubectl.ExecPodCmd(
+				namespace, appPods[helpers.App2],
+				fmt.Sprintf("curl http://%s/public", clusterIP))
```

Comment:
you should use `helpers.CurlFail(` here

Reply:
Actually  I wanted normal "curl". Fixed by removing the "curl" word